.github/workflows/numpy_1_tests.yml

```diff
@@ -0,0 +1,15 @@
+name: NumPy 1 - Tests
+on:
+
+  schedule:
+    - cron: '0 6 * * *'  # Runs daily at 6 AM UTC (1 AM Toronto in winter, 2 AM in summer)
```

Comment:
I had no idea UTC didn't include daylight-savings time! You learn something new every day...